js/showHideDateFields.js

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
@shahrukh-compuco see how we update fields based on the selected certificate type here

https://github.com/compucorp/uk.co.compucorp.certificate/blob/7d66548ebf5e19b6c5f0c477eac90de569063f8f/templates/CRM/Certificate/Form/CertificateConfigure.tpl#L64C5-L66C1

I would suggest to keep the code together.

Reply:
done